VISION.md

```diff
@@ -20,22 +20,22 @@ With these challenges in mind, [_fastlane_ plugin system](https://fabric.io/blog
 
 Each _fastlane_ tool has a specific purpose and should be kept focused on the functionality required for that task.
 
-* [deliver](https://github.com/fastlane/fastlane/tree/master/deliver): Upload screenshots, metadata, and your app binary to the App Store
```

Comment:
Tools and actions are technically not the same thing... I know before those had the same names, but now the tool named actions are only aliases. (I personally like that the "code collections" that are the tools still exist)

Reply:
I don't quite understand what should be changed here.

Reply:
The tool `deliver` (and all the others) and its code still live at Github. Only the documentation of its action moved to the docs.

Reply:
So should `VISION.md` be completely reverted? If yes, are there other files in this case?

Reply:
I personally would revert these, yes.

Reply:
I think these links go to source code of tools, not documentation of actions.